backend/engine/plugins/base_images/main.py

```diff
@@ -32,8 +32,9 @@ def find_from_lines(path: str) -> list:
             "--no-filename",
             "--recursive",
             "--include",
+            "--ignore-case",
             "Dockerfile*",
-            "^FROM (--platform=[^\\s+] ){0,1}[^\\s]+( AS .+){0,1}$",
+            "^FROM (--platform=){0,1}[-_.:\\${} /=a-z0-9]+",
```

Comment:
The new regex is missing `$` for matching the end of the line. I'm not confident that matters, but since we had it before I'd err towards keeping it